### ISSUE TITLE
Can't open site domoticz.com

### ISSUE THREAD

**Issue by kovainfo** [domoticz/domoticz#6013]
There is no way to download the update - the site is not working
https://www.domoticz.com

<img width="1360" alt="Снимок экрана 2024-02-03 в 11 39 23" src="https://github.com/domoticz/domoticz/assets/36986231/d31f06a9-7e73-4375-942f-07958984ff98">

**Comment by gizmocuz** [comment]
Unfortunately, we had a lot of spam the recent days and we had to block certain countries.
And probably your country is one of them.
(Same spam, next day via another country, next day another country and so on)
Please use a VPN and try connecting via a European country for now.
Hope to lift the block in a few days

**Comment by kovainfo** [comment]
ok, let's wait until everything calms down

**Comment by aweiand** [comment]
> Unfortunately, we had a lot of spam the recent days and we had to block certain countries. And probably your country is one of them. (Same spam, next day via another country, next day another country and so on) Please use a VPN and try connecting via a European country for now. Hope to lift the block in a few days

Seriously you are blocking an entire country? Maybe this is not the best way.... I'm receiving the same here, from Brazil, at two days sequence...

**Comment by gizmocuz** [comment]
Yes, we received many spam users/posts that were originated from different countries.
I've enable Brazil again, if it is happening again, I will add the block again.
It takes too much time to add 10 IP addresses a day each country in the block list
Maybe they already have given up...

**Comment by aweiand** [comment]
Ok, I understand you. 
Thanks for the unblock! It was working now ;-)

**Comment by kovainfo** [comment]
> Yes, we received many spam users/posts that were originated from different countries. I've enable Brazil again, if it is happening again, I will add the block again. It takes too much time to add 10 IP addresses a day each country in the block list Maybe they already have given up...

Hello! Any news about spam?
